app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt

```diff
@@ -36,9 +36,11 @@ data class SearchRequest(
     @SerializedName("free_shipping")
     var freeShipping: Int = 0,
     var location: Any = Any(),
-    var max_price: Float? = null,   // Client side casts max price and min price to an int
+    @SerializedName("max_price")
+    var maxPrice: Float? = null,   // Client side casts max price and min price to an int
                              // but the server side accepts float values
```

Comment:
Let's leave them as `Floats`. Could the comments be moved into the property kdoc?

Reply:
@scsole will do